gitcodechurn.py

```diff
@@ -100,25 +169,17 @@ def main():
             exdir
         )
 
-    # if author is empty then print a unique list of authors
-    if len(author.strip()) == 0:
-        authors = set(get_commits(before, after, author, dir, '%an')).__str__()
-        authors = authors.replace('{', '').replace('}', '').replace("'","")
-        print('authors: \t', authors)
-    else:
-        print('author: \t', author)
-    print('contribution: \t', contribution)
-    print('churn: \t\t', -churn)
-    # print files in case more granular results are needed
-    #print('files: ', files)
+    return files, contribution, churn
+    
+
 
 def get_loc(commit, dir, files, contribution, churn, exdir):
     # git show automatically excludes binary file changes
     command = 'git show --format= --unified=0 --no-prefix ' + commit
     if len(exdir) > 1:
         # https://stackoverflow.com/a/21079437
         command += ' -- . ":(exclude,icase)'+exdir+'"'
-    results = get_proc_out(command, dir).splitlines()
+    results = get_commit_results(command, dir) 
```

Comment:
@NewMountain I guess this method definition is missing..

Reply:
Same, I also get this error.

Reply:
I'm not sure how I missed this 🤦 . Let me get this sorted out and update.